server/fishtest/templates/tests_run.mak

```diff
@@ -392,13 +394,14 @@ else:
 </div>
 
 <script>
-  $(window).bind('pageshow', function() {
+   window.onpageshow = function() {
     // If pressing the 'back' button to get back to this page, make sure
     // the submit test button is enabled again.
-    $('#submit-test').removeAttr('disabled').text('Submit test');
+    document.querySelector('#submit-test').removeAttribute('disabled');
+    document.querySelector('#submit-test').innerText = 'Submit test';
     // Also make sure that the odds TC fields have the right visibility.
-    update_odds($('#checkbox-time-odds')[0]);
-  });
+    update_odds(document.querySelector('#checkbox-time-odds'));
+  };
```

Comment:
Issue: 
The scenario is if SPSA was clicked then Base branch and Base signature empty out content which is correct but when reaching the run_tests page with (Back/Forward) the stop rule enabled is SPRT then Base branch and Base signature are not refilled at all

Todo: Make sure Base branch and Base Signature are refilled when window.onpageshow

Reply:
I suggests to keep in this PR only the conversion from jQuery to vanilla js (this is valid also for the github rate limit).
When frozen, we can test the PR on PROD.